change type of noRecentsText from String to Widget (#61)
Fintasys/emoji_picker_flutter: CHANGELOG.md

## 1.2.0

> * New feature & breaking change: `noRecentsText` is now of type `Widget` (was `String` before). 
>   Inconsequence, `noRecentsStyle` was removed. This change leads to more customization options.

Reviewer: Just small suggestion to match existing format
```suggestion
- Breaking Change: `noRecentsText` is now of type `Widget` (was `String` before) and `noRecentsStyle` was removed from Config
```
Eventually need to be adjusted if property is renamed.
